src/styles/main.scss

```diff
@@ -12,3 +12,27 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media screen and  (max-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media screen and (min-width: 488px) and (max-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
Move it to `.catalog` and simplify, you don't need min and max at the same time
![image](https://user-images.githubusercontent.com/99331793/190203350-7f818447-6159-4ca6-9c76-1f1947a6423b.png)